core/src/main/java/tech/tablesaw/io/xlsx/XlsxReadOptions.java

```diff
@@ -0,0 +1,44 @@
+package tech.tablesaw.io.xlsx;
+
+import java.io.File;
+import java.io.InputStream;
+import java.io.Reader;
+
+import tech.tablesaw.io.ReadOptions;
+
+public class XlsxReadOptions extends ReadOptions {
+
+    protected XlsxReadOptions(final Builder builder) {
```

Comment:
I would suggest removing `final` everywhere it appears except for when defining class members in order to maintain consistency with the existing code base.